src/passes/referencedLibraries.ts

```diff
@@ -53,45 +73,55 @@ export class ReferencedLibraries extends ASTMapper {
   }
 }
 
+function findDeclarationInLibrary(
+  calledDeclarationId: number,
+  librariesById: Map<number, ContractDefinition>,
+): [ASTNode, ContractDefinition] | undefined {
```

Comment:
Just one small suggestion,   declaration should be of type  CallableDefinition instead of generalised ASTNode